prov/ansible/execution.go

```diff
@@ -876,6 +885,10 @@ func (e *executionCommon) generateHostConnection(ctx context.Context, buffer *by
 		if host.port != 0 && host.port != 22 {
 			buffer.WriteString(fmt.Sprintf(" ansible_ssh_port=%d", host.port))
 		}
+		// Specific ansible connection settings are needed on windows targets
+		if strings.Contains(host.osType, "windows") {
```

Comment:
Don't know but maybe it worthwhile to be mentioned somehow in the documentation.
Maybe here https://yorc.readthedocs.io/en/stable/tosca.html#supported-operations-implementations 
As it seems to be a very specific behavior.

Reply:
Ah yes, proposign to add this:

Basic support for raws scripts execution through Ansible playbooks is also provided on Windows targets configured to use ssh.

Reply:
:+1: